contracts/verification/LivepeerVerifier.sol

```diff
@@ -98,6 +107,7 @@ contract LivepeerVerifier is Manager, IVerifier {
      * @param _requestId Request identifier
      * @param _result Result of verification computation - keccak256 hash of transcoded segment data
      */
+    // solium-disable-next-line mixedcase
```

Comment:
is this a directive to the linter to ignore the warning?

Reply:
Yeah we never use mixedcase except for here because it follows the convention used by OraclizeVerifier which inherits `__callback` from the Oraclize API contract. Debatable whether we should follow that convention, but figured silencing the warning here, but enforcing the rule everywhere else would suffice for now